Source/WebCore/editing/DeleteSelectionCommand.cpp

```diff
@@ -426,7 +426,7 @@ void DeleteSelectionCommand::saveTypingStyleState()
 
     // Figure out the typing style in effect before the delete is done.
     m_typingStyle = EditingStyle::create(m_selectionToDelete.start(), EditingStyle::EditingPropertiesInEffect);
-    m_typingStyle->removeStyleAddedByNode(enclosingAnchorElement(m_selectionToDelete.start()));
```

Comment:
Oh no, don't remove this code. We need to call removeStylesAddedByNode twice. Once with enclosingAnchorElement and once with enclosingAtomicElement.

Reply:
Didn't removed. Kept it back.